### PR TITLE
Register Teaser Component

### PR DESCRIPTION
**Pull request by qamarfj**
# Description
Create a storybook for the Register Teaser Component .

Fixes # (issue)
#17 

# How to test?

Added a story on storybook for the Register Teaser Component. 
# Checklist

- [x ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/components/RegisterTeaser/RegisterTeaser.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './RegisterTeaser.css';
+import PropTypes from 'prop-types';
+import robotLogo from '../../assets/images/robot_logo.png';
+
+export default function RegisterTeaser({ singInURL }) {
```
Comment (cph-kiwi) [inline src/client/components/RegisterTeaser/RegisterTeaser.js:6]: Please change the spelling here - signInURL.

File: src/client/components/RegisterTeaser/RegisterTeaser.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  return (
+    <div className="RegisterTeaser">
+      <div>
+        <img src={robotLogo} alt="RobotLogo" />
+      </div>
+      <div className="RegisterTeaserText">
+        <p>
+          Do you want to save your results?
+          <br /> <a href={singInURL}>Sign in</a> and check your results any
```
Comment (cph-kiwi) [inline src/client/components/RegisterTeaser/RegisterTeaser.js:15]: Please change the spelling here - signInURL

File: src/client/components/RegisterTeaser/RegisterTeaser.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+          time.
+        </p>
+      </div>
+    </div>
+  );
+}
+RegisterTeaser.propTypes = {
+  singInURL: PropTypes.string.isRequired,
```
Comment (cph-kiwi) [inline src/client/components/RegisterTeaser/RegisterTeaser.js:23]: Please change the spelling here - signInURL

File: src/client/components/RegisterTeaser/RegisterTeaser.stories.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+import RegisterTeaser from './RegisterTeaser';
+
+export default {
+  title: 'Components/Register Teaser',
+  component: RegisterTeaser,
+};
+
+export const Teaser = () => <RegisterTeaser singInURL="#" />;
```
Comment (cph-kiwi) [inline src/client/components/RegisterTeaser/RegisterTeaser.stories.js:10]: Please change the spelling here - signInURL

**Review by cph-kiwi — CHANGES_REQUESTED** [review]
Looks good besides the spelling. Happy to approve after spelling change.

**Review by aijazraja — APPROVED** [review]

File: src/client/components/RegisterTeaser/RegisterTeaser.stories.js (hunk 4 above)
Comment (qamarfj) [reply src/client/components/RegisterTeaser/RegisterTeaser.stories.js]: thank you for pointing out ,I have fixed it everywhere.

**Review by cph-kiwi — APPROVED** [review]

File: src/client/components/RegisterTeaser/RegisterTeaser.css [hunk 5]
```diff
@@ -0,0 +1,20 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@500&display=swap');
```
Comment (TITHI1244) [inline src/client/components/RegisterTeaser/RegisterTeaser.css:1]: You can now delete the import as we will use from the downloaded fonts instead :)

**Review by TITHI1244 — APPROVED** [review]
Very nicely done @qamarfj. Ready to approve :)

File: src/client/components/RegisterTeaser/RegisterTeaser.css [hunk 6]
```diff
@@ -0,0 +1,18 @@
+.RegisterTeaser {
```
Comment (bytewiz) [inline src/client/components/RegisterTeaser/RegisterTeaser.css:1]:
```suggestion
.register-teaser {
```

File: src/client/components/RegisterTeaser/RegisterTeaser.css [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+  display: flex;
+  width: 50%;
+}
+
+.RegisterTeaserText p {
```
Comment (bytewiz) [inline src/client/components/RegisterTeaser/RegisterTeaser.css:6]:
```suggestion
.register-teaser-text p {
```

File: src/client/components/RegisterTeaser/RegisterTeaser.css [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+  font-family: 'Roboto Mono', monospace;
+  font-size: 15px;
+  color: #000;
+  margin-left: 3%;
+  text-align: left;
+}
+
+.RegisterTeaserText p a {
```
Comment (bytewiz) [inline src/client/components/RegisterTeaser/RegisterTeaser.css:14]:
```suggestion
.register-teaser-text p a {
```

File: src/client/components/RegisterTeaser/RegisterTeaser.js [hunk 9]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './RegisterTeaser.css';
+import PropTypes from 'prop-types';
+import robotLogo from '../../assets/images/robot_logo.png';
+
+export default function RegisterTeaser({ signInURL }) {
+  return (
+    <div className="RegisterTeaser">
```
Comment (bytewiz) [inline src/client/components/RegisterTeaser/RegisterTeaser.js:8]:
```suggestion
    <div className="register-teaser">
```

File: src/client/components/RegisterTeaser/RegisterTeaser.js [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+      <div>
+        <img src={robotLogo} alt="RobotLogo" />
+      </div>
+      <div className="RegisterTeaserText">
```
Comment (bytewiz) [inline src/client/components/RegisterTeaser/RegisterTeaser.js:12]:
```suggestion
      <div className="register-teaser-text">
```

**Review by bytewiz — CHANGES_REQUESTED** [review]
Just missing to update the classnames to kebab-case

**Review by bytewiz — APPROVED** [review]

**Pull request merged**